app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] Normally, it is a good practice to verify your application users before letting them use some features of your application. The reason is that you don't want fake users. A remedy to this problem is to add `confirmable` to here and ensure that you send an email to the newly created user.

Reply:
thanks i just did this, but how do I delete those unused files? because if I delete the vendor folder the application breaks